docs/administer/rbac.md

```diff
@@ -0,0 +1,435 @@
+
+        ```sh
+        p, team-darkside:role, namespaces, read, */*
+        p, team-darkside:role, database-engines, read, */*
+        p, team-darkside:role, database-clusters, read, */*
+        p, team-darkside:role, database-clusters, update, */*
+        p, team-darkside:role, database-clusters-credentials, read, */*
+        p, team-darkside:role, backup-storages, read, */*
+        p, team-darkside:role, monitoring-instances, read, */*
+        ```
+
+        ??? info "Let's dive into decoding this!"
+
+            The `team-darkside` role has the following privileges in **all** the namespaces:
+
+            - **namespace**:  `Read` access in all the namespaces.
+            - **Database engines**: `Read` access to all the database engines
+            - **Database clusters**:  `Read` and update access for all the database clusters
+            - **Database clusters credentials**: `Read` acccess for **all** the databases
+            - **Backup storages**: `Read` access to all the backup storages
+            - **Monitoring instances**: `Read` access to all the monitoring instances
+
+
+## Assigning roles to users
+
+In order for roles to take effect, they need to be assigned to users. The syntax for this is as follows:
+
+```sh
+g, username, rolename
+```
+
+A new user in Percona Everest will initially have **no** permissions. To grant permissions, you must edit your RBAC configuration stored in the `everest-rbac` `ConfigMap` in the `everest-system` namespace:
+
+
+```sh
+kubectl edit configmap-rbac -n everest-system
+```
+
+A text editor will open, and you can edit the `ConfigMap` as follows. You just have to add the new user and assign it the desired role.
+
+
+```sh
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: everest-rbac
+data:
+  policy.csv: |
+    p, adminrole:role, namespaces, *, *
+    p, adminrole:role, database-engines, *, */*
+    p, adminrole:role, database-clusters, *, */*
+    p, adminrole:role, database-cluster-backups, *, */*
+    p, adminrole:role, database-cluster-restores, *, */*
+    p, adminrole:role, database-cluster-credentials, *, */*
+    p, adminrole:role, backup-storages, *, */*
+    p, adminrole:role, monitoring-instances, *, */*
+    g, admin, adminrole:role
+    g, <newuser>, adminrole:role
+```
+
+For example, let's assign the role `team-dev` to a new user named John:
+
+```sh       
+p, team-dev:role, namespaces, read, dev
+p, team-dev:role, database-engines, read, dev/*
+p, team-dev:role, database-clusters, read, dev/*
+p, team-dev:role, database-clusters, update, dev/*
+p, team-dev:role, database-clusters, create, dev/*
+p, team-dev:role, database-clusters, delete, dev/*
+p, team-dev:role, database-cluster-credentials, read, dev/*
+p, team-dev:role, backup-storages, read, dev/*
+p, team-dev:role, monitoring-instances, read, dev/*
+p, team-dev:role, database-cluster-restores, create, dev/*
+g, john, team-dev:role
```

Comment:
Let's keep it simple for now, no backups/restores
```suggestion
p, role:team-dev, namespaces, read, dev
p, role:team-dev, database-engines, read, dev/*
p, role:team-dev, database-clusters, read, dev/*
p, role:team-dev, database-clusters, update, dev/*
p, role:team-dev, database-clusters, create, dev/*
p, role:team-dev, database-clusters, delete, dev/*
p, role:team-dev, database-cluster-credentials, read, dev/*
g, john, role:team-dev
```

Reply:
I'm actually questioning this now, should this even be here? Should we simply move this to the examples section?

Reply:
I think we should... In this section we just need to show how to actually assign a user with we already did with the `role:admin` example above. I think this makes it too crowded.